src/redux/user/user.reducer.js

```diff
@@ -26,7 +27,8 @@ export const initialState = {
   passwordReset: false,
   confirmationEmailSent: false,
   recoveryLoading: false,
-  confirmationLoading: false
+  confirmationLoading: false,
+  emailError: null
```

Comment:
that is not a good idea to add an additional error field

Reply:
fixed